packages/mailgun/test/index.js

```diff
@@ -5,7 +5,7 @@ const { execute, send } = Adaptor;
 import { fields, field } from '@openfn/language-common';
 
 describe('execute', () => {
-  it('sends an Email and expects a confirmation response', done => {
+  it.skip('sends an Email and expects a confirmation response', done => {
```

Comment:
please open the issue to fix this before we merge this PR in

Reply:
This was actually on me to do! 

I'm doing that now. I just stopped by this PR to reference it :)